src/effects/effectslot.cpp

```diff
@@ -179,8 +181,6 @@ void EffectSlot::loadEffect(EffectPointer pEffect) {
             pParameter->loadEffect(pEffect);
         }
 
-        slotEffectMetaParameter(m_pControlMetaParameter->get(), true);
```

Comment:
Ah OK, I have not noticed. But this will not solve the issue where the user has no metaknob. 

I wish to disable the effects, select a desired effect, enable the effects and it should play at its default. If I do not like the defaults, I can turn the metaknob to the desired position before I enable the effect and the new value will be immediately adopted.  

I concider the current metaknob position during effect load as a perfect position for the previous loaded effect but a random position for the new effect. So the defaults are in any case better than that. If not turning the metaknob is required anyway. 

This is almost the same behavior we have in 2.0 with the super knob, precept we have now the bypass that allows to adopt the new value by jump when the effect is disabled. 

We may improve this issue later by a default value of the meatknob like 0.5 that matches artificially the default position of the effect. This is done for example at a Pioneer Hardware Mixxx. We have discussed this as well when introducing the Quick effect knob. I still support the idea, but it was rejected because it requires a special effect design or it will lead only to one usable side of the knob, or likewise the same behaviour on both knob sides.